package.json

```diff
@@ -41,7 +41,6 @@
     "@typescript-eslint/eslint-plugin": "^4.29.2",
     "eslint": "7.32.0",
     "eslint-config-next": "11.1.0",
-    "prisma": "2.23.0",
```

Comment:
why do we remove prisma from dependencies here please?

Reply:
my bad